PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetQnVectors.h

```diff
@@ -0,0 +1,101 @@
+#ifndef ALIANALYSISTASKJETQNVECTORS_H
+#define ALIANALYSISTASKJETQNVECTORS_H
+
+/* Copyright(c) 1998-2019, ALICE Experiment at CERN, All rights reserved. *
+ * See cxx source for full Copyright notice                               */
+
+/* $Id$ */
+
+//*************************************************************************************************
+// \class AliAnalysisTaskJetQnVectors
+// \brief task used to load the Qn calibrations and get the calibrated Qn vectors for HF analyses
+// \authors:
+// F. Grosa, fabrizio.grosa@cern.ch
+// F. Catalano, fabio.catalano@cern.ch
+// A. Dobrin, alexandru.dobrin@cern.ch
+// A. Festanti, andrea.festanti@cern.ch
+// G. Luparello, grazia.luparello@cern.ch
+// F. Prino, prino@to.infn.it
+// A. Rossi, andrea.rossi@cern.ch
+// S. Trogolo, stefano.trogolo@cern.ch
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <TH1F.h>
+#include <TList.h>
+#include <TString.h>
+#include <TMath.h>
+
+#include "AliAnalysisTaskSE.h"
+#include "AliVEvent.h"
+#include "AliAODEvent.h"
+#include "AliJEQnVectorHandler.h"
+
+using namespace std;
```

Comment:
Please move this to your .cxx. `using namespace std` is generally to be avoided in a header